### ISSUE TITLE
Feature Request: Allow for choosing custom font for responses and code blocks

### ISSUE THREAD

**Issue by eshack94** [josStorer/chatGPTBox#169]
**Is your feature request related to a problem? Please describe.**

Ability to change font used by chatGPTBox, especially code font. I would like to be able to use a custom system font.

**Describe the solution you'd like**

I want to be able to use custom fonts for chatGPTBox.

Add custom font menu available in settings. Allow 2 fonts to be chosen. One font for text responses, and another font for code blocks.

**Additional context**

N/A

**Comment by josStorer** [comment]
OK

**Comment by eshack94** [comment]
@josStorer just curious — is this still on the roadmap?

**Comment by josStorer** [comment]
I still have plans to implement this feature, but currently it's not a high priority. Recently, I'm working on [RWKV-Runner](https://github.com/josStorer/RWKV-Runner), so updates for chatGPTBox have slowed down.

**Comment by eshack94** [comment]
Makes sense! No rush, was just curious if you were still considering it.

Looking forward to your new project!
